src/__tests__/pages/FestivalsPage.test.tsx

```diff
@@ -32,15 +46,33 @@ const TestWrapper = ({ children }: { children: React.ReactNode }) => {
 
 describe('FestivalsPage 테스트', () => {
   vi.spyOn(ReactRouter, 'useParams').mockReturnValue({ areaId: '2' });
-  describe('페이지 기본 구조', () => {
-    test('스냅샷 테스트', () => {
-      // 기본 스냅샷 테스트
+  describe('스냅샷 테스트', () => {
+    test('AI 추천 섹션 스냅샷', async () => {
       const { container } = render(
         <TestWrapper>
           <FestivalsPage />
         </TestWrapper>,
       );
-      expect(container.firstChild).toMatchSnapshot();
+
+      // Suspense가 끝나고 섹션이 나타날 때까지 기다림
+      await waitFor(() => {
+        const sections = container.querySelectorAll('section');
+        expect(sections.length).toBeGreaterThanOrEqual(2);
+      });
+
+      const sections = container.querySelectorAll('section');
+
+      // 0번: AI, 1번: Festivals 라는 현재 구조에 의존
+      const aiPickSection = sections[0];
+      const festivalsSection = sections[1];
```

Comment:
```suggestion
      const [aiPickSection, festivalsSection] = sections;
```

이건 syntactic sugar 인데 이렇게 간결하게 표현하는 방법도 있습니다.

하드코딩과 진배없지만 하드 코딩을 안 한 것처럼 보일 수 있는 방법이죠.